ocs_ci/ocs/resources/pod.py

```diff
@@ -2416,3 +2416,19 @@ def wait_for_osd_pods_having_ids(osd_ids, timeout=180, sleep=10):
         if len(osd_pods) == len(osd_ids):
             logger.info(f"Found all the osd pods with the ids: {osd_ids}")
             return osd_pods
+
+
+def pod_resource_utilization_raw_output_from_adm_top(
```

Comment:
Please keep the convention and add "Args" & "Returns"

Reply:
Done